src/test/ui/lint/noop-method-call.stderr

```diff
@@ -0,0 +1,39 @@
+warning: call to `.clone()` on a reference in this situation does nothing
+  --> $DIR/noop-method-call.rs:24:32
+   |
+LL |     let foo_clone: &Foo<u32> = foo.clone();
+   |                                ^^^^^^^^^^^ unnecessary method call
```

Comment:
Could we point to just the method call instead (even if we don't provide a suggestion)?